Assignment/solution_student/css/style.css

```diff
@@ -31,19 +31,120 @@ html {
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
+  color: #121212;
   font-size: 16px;
-  background: white;
+  background-color: #f0efef;
+  font-weight: 300px;
+  font-family: 'open Sans', sans-serif;
+
 }
 
 /******************************************
 /* LAYOUT
 /*******************************************/
 
+
 /******************************************
 /* PAGES
 /*******************************************/
 
 /******************************************
 /* SECTIONS
-/*******************************************/
\ No newline at end of file
+/*******************************************/
+
+/* Site Header */
+header {
+  background-color: #033048;
+  height: 100px;
+  padding: 40px 0;
+  
+
+}
+header h1{
+	display: inline-block;
+	font-size: 24px;
+	margin: 0;
+	color: #f0efef;
+	margin: 0;
+	padding-left: 15%;
+	float: left;
+}
+nav ul{
+	padding-right: 10%;
+	padding-top: 1em;
+	margin: 0px;
+	float: right;
+}
+nav li{
+	display: inline-block;
+
+	font-weight: 500;
+	color: #f9e42e;
+	margin: 0px
+}
+a{
+	color: yellow;
+	text-decoration: none;
+}
+
+#adUnit{
+	background: url('images/ad-unit.png');
+    /*background-size: cover;*/
```

Comment:
Couple things going on here:
1) Your relative url is off — what you've written would be looking for the ad-unit.png file inside of an 'images' folder that lives inside the current folder, which from the CSS document is in the '/css' folder, so it would com out: `css/images/ad-unit.png`, which doesn't exist as you know. You want to travel up one level out of the 'css' folder, which is where the reference is being made in the style.css, and then back down into the '/images' folder: `../images/ad-unit.png` Refer back to lesson 02 for help: https://github.com/GA-Chicago-FEWD6/Week_01_Basics/blob/master/02_css_basics/slides.md
2) You are setting the image as a background image, but your #adUnit has no inherent dimension, it's just an empty div, so if you want the image to show up you need to give it some dimension: `width: 240px; height: 546px;` for example — in this situation I think it'd be easier and make more sense to just include the image as an `img` tag in the html instead of a background image though